documentation/api.html

```diff
@@ -18,7 +18,7 @@
title": "Invite Link"}}, "type": "object", "required": ["invite_link"], "title": "InviteLink", "description": "Response data for the `/bot/create` endpoint."}, "Response": {"properties": {"msg": {"type": "string", "title": "Msg", "default": "Success"}}, "type": "object", "title": "Response", "description": "Simple response model. Note that for errors, it's better to raise\nan HTTPException."}, "Role": {"properties": {"name": {"type": "string", "title": "Name"}, "id": {"anyOf": [{"type": "integer"}, {"type": "null"}], "title": "Id"}, "permissions": {"items": {"type": "string"}, "type": "array", "title": "Permissions", "default": []}}, "type": "object", "required": ["name"], "title": "Role", "description": "A role for a server. Has some permissions."}, "ServerConfig-Input": {"properties": {"name": {"type": "string", "title": "Name"}, "id": {"anyOf": [{"type": "integer"}, {"type": "null"}], "title": "Id"}, "community": {"type": "boolean", "title": "Community", "default": false}, "roles": {"items": {"$ref": "#/components/schemas/Role"}, "type": "array", "title": "Roles", "default": []}, "categories": {"items": {"$ref": "#/components/schemas/Category-Input"}, "type": "array", "title": "Categories", "default": []}}, "type": "object", "required": ["name"], "title": "ServerConfig", "description": "A configuration for a single server."}, "ServerConfig-Output": {"properties": {"name": {"type": "string", "title": "Name"}, "id": {"anyOf": [{"type": "integer"}, {"type": "null"}], "title": "Id"}, "community": {"type": "boolean", "title": "Community", "default": false}, "roles": {"items": {"$ref": "#/components/schemas/Role"}, "type": "array", "title": "Roles", "default": []}, "categories": {"items": {"$ref": "#/components/schemas/Category-Output"}, "type": "array", "title": "Categories", "default": []}}, "type": "object", "required": ["name"], "title": "ServerConfig", "description": "A configuration for a single server."}, "Token": {"properties": {"access_token": {"type": "string", "title": "Access Token"}, "token_type": {"type": "string", "title": "Token Type"}}, "type": "object", "required": ["access_token", "token_type"], "title": "Token", "description": "Login tokens. `token_type` should be `bearer`."}, "User": {"properties": {"username": {"type": "string", "title": "Username"}, "servers": {"items": {"type": "string"}, "type": "array", "title": "Servers", "default": []}}, "type": "object", "required": ["username"], "title": "User", "description": "Data about a user. `servers` is a list of server names."}, "ValidationError": {"properties": {"loc": {"items": {"anyOf": [{"type": "string"}, {"type": "integer"}]}, "type": "array", "title": "Location"}, "msg": {"type": "string", "title": "Message"}, "type": {"type": "string", "title": "Error Type"}}, "type": "object", "required": ["loc", "msg", "type"], "title": "ValidationError"}}, "securitySchemes": {"OAuth2PasswordBearer": {"type": "oauth2", "flows": {"password": {"scopes": {}, "tokenUrl": "auth/login"}}}}}};
```

Comment:
consider mentioning that the invite link returned from bot/create is the link to join the newly created server on Discord

Reply:
👍  see 7dd2c9a